unit_test/raw_image_test.py

```diff
@@ -0,0 +1,138 @@
+# Fluster - testing framework for decoders conformance
+# Copyright (C) 2021, Fluendo, S.A.
+#  Author: Manuel Jimeno Martínez <mjimeno@fluendo.com>, Fluendo, S.A.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Library General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Library General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public
+# License along with this library; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
+import unittest
+from typing import List
+from fluster.raw_image import RawImage, RawBufferType, ComparisonType
+
+
+class SyntheticImage:
+    """Generate a synthetic RGBA image"""
+
+    RED = [0xFF, 0, 0, 0xFF]
+    GREEN = [0, 0xFF, 0, 0xFF]
+    BLUE = [0, 0, 0xFF, 0xFF]
+
+    def __init__(
+        self,
+        first_band: List[int],
+        second_band: List[int],
+        third_band: List[int],
+        color_band_width: int,
```

Comment:
This naming scheme is confusing to me. Can you rename or comment the code please?